portfolio.css

```diff
@@ -0,0 +1,1512 @@
+    overflow: hidden;
+    background-image: url(./images/Placeholder3.png);
+  }
+
+  .profesional1 {
+    display: none;
+  }
+
+  .profesional2 {
+    display: none;
+  }
+
+  .profetional2-desktop {
+    display: flex;
+    flex-direction: column;
+    background-image: url(./images/Placeholder4.png);
+  }
+
+  .profesional3 {
+    display: none;
+  }
+
+  .profetional3-desktop {
+    display: flex;
+    flex-direction: column;
+    background-image: url(./images/Placeholder5.png);
+  }
+
+  .profesional4 {
+    display: none;
+  }
+
+  .profesional4-desktop {
+    display: flex;
+    flex-direction: column;
+    background-image: url(./images/Placeholder1.png);
+  }
+
+  .profesional5 {
+    display: none;
+  }
+
+  .profesional6 {
+    display: none;
+  }
+
+  .buttons1 > ul {
+    align-self: flex-end;
+  }
+
+  .section3 {
+    width: 100vw;
+    height: 1401px;
+  }
+
+  .aboutMe {
+    margin-top: 100px;
+    overflow: hidden;
+  }
+
+  .aboutMeLower {
+    position: absolute;
+    left: 85.94%;
+    right: 77.22%;
+    bottom: 90.67%;
+  }
+
+  .me {
+    margin-top: 44px;
+    margin-left: 60px;
+  }
+
+  .hello {
+    margin-left: 60px;
+    margin-top: 80px;
+  }
+
+  .get {
+    margin-left: 60px;
+  }
+
+  .lastImage {
+    display: none;
+  }
+
+  .big-image {
+    display: flex;
+    position: relative;
+    width: 40%;
+    transform: matrix(1, 0, 0, 1, -350, -100);
+    margin-bottom: -6400px;
+    margin-top: 300px;
+    margin-left: 44px;
+  }
+
+  .devider-image {
+    display: flex;
+    width: 1200px;
+    margin-top: -284px;
+  }
+
+  .abilities {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    margin-bottom: 180px;
+    margin-top: 80px;
+  }
+
+  .language {
+    position: relative;
+  }
+
+  .language ul {
+    margin-top: -30px;
+  }
+
+  .frameworks {
+    position: relative;
+    margin-top: -200px;
+  }
+
+  .skills {
+    position: relative;
+    margin-top: -480px;
+  }
+
+  .skills ul {
+    margin-top: 15px;
+  }
+
+  .lastimage {
+    position: relative;
+    left: 90%;
+    right: 80.45%;
+    top: 84.33%;
+    bottom: -25.14%;
+  }
+
+  .section4 {
+    width: 100vw;
+    margin-bottom: 200px;
+  }
+
+  .section4Top {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    margin-bottom: 40px;
+  }
+
+  .always {
+    margin-top: 60px;
+    margin-left: 80px;
+  }
+
+  .forms {
+    display: flex;
+    flex-direction: column;
+    width: 30%;
+    overflow: hidden;
+    margin-left: 100px;
+  }
+
+  .input {
+    height: 18px;
+  }
+
+  .images-last {
+    width: 100vw;
+    margin-top: -100px;
+    overflow: hidden;
+  }
+
+  .touch {
+    margin-left: -50px;
+  }
+
+  .social-media1 > ul {
+    align-items: center;
+    margin-left: 36%;
+  }
+
+  .shape {
+    display: none;
+  }
+
+  .desktop-popup {
+    display: none;
+    position: relative;
+    width: 82%;
+    padding-left: 8%;
+    padding-right: 8%;
+    margin-bottom: -20%;
+    z-index: 99;
+  }
```

Comment:
- [ ] `required`
- [ ] Good job so far!! :clap: :clap: 
- [ ] Kindly make the `pop-up cards` as similar as possible to the figma template.
- [ ] Please see to it that if someone clicks on see project, it pops up right there, and not else where
- [ ] also see to it the `fonts` `margins` and `responsivements` is working fine
- [ ] please see screenshots below
- `your template`
![Screenshot from 2022-07-23 00-07-00](https://user-images.githubusercontent.com/66230306/180575502-470159ad-069f-44ed-90f1-5dfd6b5eae67.png)

- `figma template`
![Screenshot from 2022-07-23 00-21-03](https://user-images.githubusercontent.com/66230306/180577244-0c16f86b-b89a-4d24-9267-52ee63201876.png)